### PR TITLE
feat: challenge 46 Simple Animations

### PR DESCRIPTION
**Pull request by svenson95**
Created new challenge about the integrated `@angular/animations` api from angular

### CODE REVIEW

File: apps/angular/simple-animations/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Simple Animations
+
+> author: thomas-laforge
```
Comment (jdegand) [inline apps/angular/simple-animations/README.md:3]: Need your name here.

File: apps/angular/simple-animations/README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+### Run Application
+
+```bash
+npx nx serve angular-simple-animations
+```
+
+### Documentation and Instruction
+
+Challenge documentation is [here](https://angular-challenges.vercel.app/challenges/angular/45-simple-animations/).
```
Comment (jdegand) [inline apps/angular/simple-animations/README.md:13]: Challenge number needs to be 46.

File: challenge-number.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "total": 45,
```
Comment (jdegand) [inline challenge-number.json:2]: update total to 46.

File: docs/src/content/docs/challenges/angular/45-simple-animations.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
+---
+title: 🟢 Simple Animations
+description: Challenge 45 is about learning angular's integrated animation API
```
Comment (jdegand) [inline docs/src/content/docs/challenges/angular/45-simple-animations.md:3]: 46

File: docs/src/content/docs/es/index.mdx [hunk 5]
```diff
@@ -13,7 +13,7 @@ hero:
       icon: right-arrow
       variant: primary
     - text: Ir al Desafío más reciente
-      link: /es/challenges/angular/43-signal-input/
+      link: /es/challenges/angular/45-simple-animations/
```
Comment (jdegand) [inline docs/src/content/docs/es/index.mdx:16]: 46

File: docs/src/content/docs/fr/index.mdx [hunk 6]
```diff
@@ -15,7 +15,7 @@ hero:
       icon: right-arrow
       variant: primary
     - text: Aller au dernier Challenge
-      link: /fr/challenges/angular/44-view-transition/
+      link: /fr/challenges/angular/45-simple-animations/
```
Comment (jdegand) [inline docs/src/content/docs/fr/index.mdx:18]: 46

File: docs/src/content/docs/pt/index.mdx [hunk 7]
```diff
@@ -13,7 +13,7 @@ hero:
       icon: right-arrow
       variant: primary
     - text: Ir para o desafio mais recente
-      link: /pt/challenges/angular/44-view-transition/
+      link: /pt/challenges/angular/45-simple-animations/
```
Comment (jdegand) [inline docs/src/content/docs/pt/index.mdx:16]: 46

File: docs/src/content/docs/ru/index.mdx [hunk 8]
```diff
@@ -13,7 +13,7 @@ hero:
       icon: right-arrow
       variant: primary
     - text: Перейти к последней задаче
-      link: /ru/challenges/angular/44-view-transition/
+      link: /ru/challenges/angular/45-simple-animations/
```
Comment (jdegand) [inline docs/src/content/docs/ru/index.mdx:16]: 46

File: docs/src/content/docs/challenges/angular/45-simple-animations.md [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+  badge: New
+---
+
+## Information
+
+Well-designed animations can make your application more fun and straightforward to use, but they aren't just cosmetic. Animations can improve your application and user experience in a number of ways:
+
+- Without animations, web page transitions can seem abrupt and jarring
+- Motion greatly enhances the user experience, so animations give users a chance to detect the application's response to their actions
+- Good animations intuitively call the user's attention to where it is needed
```
Comment (jdegand) [inline docs/src/content/docs/challenges/angular/45-simple-animations.md:18]: add periods after every sentence.

File: docs/src/content/docs/challenges/angular/45-simple-animations.md [hunk 10]
```diff
@@ -0,0 +1,38 @@
...
+
+Don't use any css and utilize angular's integrated `@angular/animations` api.
+
+### Level 1
+
+Add a fading or moving animation for the paragraphs on the left side
+
+### Level 2
+
+Add a stagger animation for the list on the right side
```
Comment (jdegand) [inline docs/src/content/docs/challenges/angular/45-simple-animations.md:38]: periods after sentences on level 1 & 2.

File: docs/src/content/docs/challenges/angular/45-simple-animations.md [hunk 11]
```diff
@@ -0,0 +1,38 @@
...
+
+## Information
+
+Well-designed animations can make your application more fun and straightforward to use, but they aren't just cosmetic. Animations can improve your application and user experience in a number of ways:
+
+- Without animations, web page transitions can seem abrupt and jarring
+- Motion greatly enhances the user experience, so animations give users a chance to detect the application's response to their actions
+- Good animations intuitively call the user's attention to where it is needed
+
+I would recommend you to read the [official documentation](https://angular.io/guide/animations). You will learn everything that is necessary to successfully complete the challenge.
```
Comment (jdegand) [inline docs/src/content/docs/challenges/angular/45-simple-animations.md:20]: remove `to`

File: docs/src/content/docs/challenges/angular/45-simple-animations.md [hunk 12]
```diff
@@ -0,0 +1,38 @@
...
+
+Otherwise look at this [working example](https://svenson95.github.io/ng-xmp-animations/) and [git repo](https://github.com/svenson95/ng-xmp-animations) to get inspired.
+
+## Statement
+
+The goal of this challenge is to add animations.
+
+## Constraints
+
+Don't use any css and utilize angular's integrated `@angular/animations` api.
```
Comment (jdegand) [inline docs/src/content/docs/challenges/angular/45-simple-animations.md:30]: Capitalize CSS, Angular, and API.

File: docs/src/content/docs/challenges/angular/45-simple-animations.md (hunk 4 above)
Comment (jdegand) [reply docs/src/content/docs/challenges/angular/45-simple-animations.md:3]: Captialize Angular

**Review by jdegand — COMMENTED** [review]
I'd add a comment about how this challenge will help someone later complete the `view-transition` challenge.   

I need to look into Angular animations more and I think you understate some of the benefits.

File: apps/angular/simple-animations/README.md (hunk 2 above)
Comment (jdegand) [reply apps/angular/simple-animations/README.md:13]: I think the vercel build failed cause of the number.

File: docs/src/content/docs/challenges/angular/45-simple-animations.md (hunk 11 above)
Comment (svenson95) [reply docs/src/content/docs/challenges/angular/45-simple-animations.md:20]: isn't that correct? you mean the first "to" in this? (... recommend you **to** read the ...)

**Comment by svenson95** [comment]
@jdegand there was some other places where i had to update the challenge number as well, please check the changes in this [commit](https://github.com/tomalaforge/angular-challenges/pull/612/commits/070f0ff6ee5aedb102c3b530a41f333044fde784)

**Comment by tomalaforge** [comment]
The build is failing because you need to add a file with your name inside the author folder.
(I need to update the doc or the schematic for this), that's a common mistake

Did you use the schematic to generate your challenge?

Great challenge, I wanted to do an animation one, so that's perfect, thanks a lot. 

I'll review it when I have some time, but thanks @jdegand for doing a review.,

File: challenge-number.json (hunk 3 above)
Comment (tomalaforge) [reply challenge-number.json]: I need to create a new schematic to update a challenge to another number.

**Comment by svenson95** [comment]
@tomalaforge yes i used the generator, like explained in the _Create your own challenge_ article.

File: docs/src/content/docs/challenges/angular/45-simple-animations.md (hunk 11 above)
Comment (jdegand) [reply docs/src/content/docs/challenges/angular/45-simple-animations.md]: `I would recommend you read` is correct and flows better.  `to` is unnecessary.  Sorry, should I clarified better.

Comment (svenson95) [reply docs/src/content/docs/challenges/angular/45-simple-animations.md]: done thx for your feedback!

File: docs/src/content/docs/challenges/angular/46-simple-animations.md [hunk 13]
```diff
@@ -0,0 +1,40 @@
...
+
+Don't use any CSS and utilize Angular's integrated `@angular/animations` API.
+
+### Level 1
+
+Add a fading or moving animation for the paragraphs on the left side.
+
+### Level 2
+
+Add a stagger animation for the list on the right side.
```
Comment (tomalaforge) [inline docs/src/content/docs/challenges/angular/46-simple-animations.md:40]: I would to have a video on how to final result should look like. It's easier to understand what we should do

Comment (svenson95) [reply docs/src/content/docs/challenges/angular/46-simple-animations.md:40]: Yes you're right, i'll update the docs to make this more understandable. 

Thanks for creating this project! The challenges are really nice and helpful, hope i can contribute more in future 😊

Comment (svenson95) [reply docs/src/content/docs/challenges/angular/46-simple-animations.md:40]: changes done!

Comment (tomalaforge) [reply docs/src/content/docs/challenges/angular/46-simple-animations.md:44]: Ok that's way better. 👍

But for the video, if you have already done the challenge, you could show the final result by recording your screen. 
But that's fine for me. I'll merge it tomorrow.
And then, you can submit your solution in one or two weeks.

Great job. Thanks a lot

Comment (svenson95) [reply docs/src/content/docs/challenges/angular/46-simple-animations.md:44]: Perfect. I think it's better if we don't show the final result in advance. As soon as this PR is merged i'll build my solution and add a final result video in the solution pull request.

Comment (tomalaforge) [reply docs/src/content/docs/challenges/angular/46-simple-animations.md:44]: I disagree on this point. Having the final result is always better. 
Wen you work in a company, the design team or product team shows the desired result in a mock template like figma and you can implement it afterward. 
But I will not block the PR for it.

Comment (svenson95) [reply docs/src/content/docs/challenges/angular/46-simple-animations.md:44]: Okay then i'll add the final result to the docs when i've got the solution done. Thanks for the feedback :)

**Comment by tomalaforge** [comment]
@svenson95 I will merge it next week. I haven't got the time this week. Prod was on fire at my work 😅

**Review by tomalaforge — APPROVED** [review]

**Comment by tomalaforge** [comment]
@all-contributors add @svenson95 for doc

**Pull request merged**